Apps/WebClient/src/ClientApp/src/views/HomeView.vue

```diff
@@ -281,6 +295,13 @@ function handleClickOrganDonorCard(): void {
     router.push({ path: "/services" });
 }
 
+function handleClickHealthConnectCard(): void {
+    trackClickLink("home_primarycare");
+    window.location.replace(
+        "https://www.healthlinkbc.ca/health-connect-registry"
+    );
```

Comment:
This will not preserve history, and it will navigate away from the site.

I suggest switching to `window.open()` to open the link in a new tab, similar to other external links.

Reply:
my computer froze in the middle of changing it to window.open("") to open a new tab. good catch.